p2p/src/main/java/bisq/network/p2p/storage/P2PDataStorage.java

```diff
@@ -121,6 +124,8 @@
     private final Set<ProtectedDataStoreListener> protectedDataStoreListeners = new CopyOnWriteArraySet<>();
     private final Clock clock;
 
+    protected int maxSequenceNumberMapSizeBeforePurge;
```

Comment:
can we have a 
```suggestion
    protected static final int MAX_SEQUENCE_NUMBER_MAP_SIZE_BEFORE_PURGE;
```
or at least 
```suggestion
    protected final int maxSequenceNumberMapSizeBeforePurge;
```
?

or as I am sure you thought about that, why did you decide against that in the first place?

Reply:
https://github.com/julianknutsen/bisq/commit/850cd9180213f396180b447d6164e7e8c3900ac2

Guice is really new to me, but from what I found I think this solution will be better. The App will just inject the static value and the tests can pass it in through the normal constructor.

The constant didn't feel right to put in NetworkOptionKeys so let me your preference for it.

Let me know your comments and I'll create the PR to fix this up.

Reply:
I agree with the changes you made in the commit you referenced.